client/my-sites/plugins/marketplace/marketplace-plugin-setup-status/index.tsx

```diff
@@ -44,10 +110,10 @@ function WrappedMarketplacePluginSetup( {
 			<div className="marketplace-plugin-setup-status__root">
 				<div>
 					<h1 className="marketplace-plugin-setup-status__title wp-brand-font">
-						Installing Plugin { pluginStatus }
+						{ <div>{ currentStage }</div> }
```

Comment:
Did we mean to have a `div` inside a `h1`?

Reply:
Fixed by c8f063a